### DISCUSSION TITLE
Image Download

### DISCUSSION THREAD

**Discussion by jackshaw20** [bimmerconnected/bimmer_connected#711] [General]
I am trying to download vehicle images to used in HA,   I am getting this error on a CLI prompt.  any help would be much appreciated 

![image](https://github.com/user-attachments/assets/8eec366f-7bdc-41cc-b248-7c471f581da8)

**Comment by rikroe** [comment]
Can you please open a separate issue?
This is an actual bug.

Other than that, I am working on a extension to HA to show the images natively.

**Comment by rikroe** [comment]
The error is caused as you are using Python 3.8.
Apparently, this is the only function that doesn't work there.

As 3.8 is end of live (see https://devguide.python.org/versions/) I won't fix this but rather drop support in total.

Please use a newer python version.

**Reply by jackshaw20** [reply]
ok thanks, I will download the new version and try again,  how do I select what view or does this download all?

would be good to be native in HA :)

**Reply by rikroe** [reply]
All views will be downloaded.
